django_mailbox/models.py

```diff
@@ -598,6 +611,8 @@ def set_body(self, body):
 
     def get_email_object(self):
         """ Returns an `email.message.Message` instance for this message."""
+        if self.eml:
+            return email.message_from_file(Message.objects.last().eml)
```

Comment:
`Message.objects.last()` is probably not what you were looking for -- that'll always fetch the last `Message` object in the queryset.  You *probably* meant to instead do something like...

```python
if self.eml:
    return email.message_from_file(self.eml)
```

Also, `.last()` is only supported in relatively modern (1.6+) versions of Django, and we do try to keep support for versions as old as 1.4.x for now.